packages/ui/src/components/SliderSteps/SliderSteps.test.tsx

```diff
@@ -1,15 +1,79 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
+import React, { useState } from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
 
 import { SliderSteps } from './SliderSteps';
 
 describe('SliderSteps', () => {
+  const givenStepsCount = 5;
+  const lastDotIndex = givenStepsCount - 1;
+  const selectedIndex = 2;
+
+  const TestSliderComponent = ({ startSelectedIndex }: { startSelectedIndex: number }) => {
+    const [selected, setSelectedIndex] = useState(startSelectedIndex);
+
+    return <SliderSteps count={givenStepsCount} selectedIndex={selected} onChange={setSelectedIndex} />;
+  };
```

Comment:
💭 It would be better to test only the component - not its implementation. In practical terms it means you render normal SliderSteps, mock the `onChange` function and check what argument was passed to it.

Reply:
Why you couldn't test such a case? I think you can use `expect(onChange).not.toHaveBeenCalled()`